components/AppBar/src/MDCAppBar.m

```diff
@@ -220,6 +220,7 @@ - (UIBarButtonItem *)backButtonItem {
                                                         action:@selector(didTapBackButton:)];
   }
   backBarButtonItem.accessibilityIdentifier = @"back_bar_button";
+  backBarButtonItem.accessibilityLabel = NSLocalizedString(@"Back", @"Back");
```

Comment:
Won’t you need a strings file in the AppBar component? Otherwise this will never be localized.